recipe/meta.yaml

```
@@ -1,63 +1,96 @@
-{% set version = "0.9.0" %}
+{% set name = "geopandas" %}
+{% set version = "0.12.2" %}
 
 package:
-  name: geopandas
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/geopandas/geopandas/releases/download/v{{ version }}/geopandas-{{ version }}.tar.gz
-  sha256: 63972ab4dc44c4029f340600dcb83264eb8132dd22b104da0b654bef7f42630a
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0acdacddefa176525e4da6d9aeeece225da26055c4becdc6e97cf40fa97c27f4
 
 build:
-  number: 1
-  noarch: python
+  number: 0
+  # rtree is missing on s390x.
+  # geopandas has unsatisfable dependencies for python 3.10 (Seems like the incompatible dependencies are
+  # libgdal-3.0.2-h0f233a1_0 requires proj >=6.2.1,<6.2.2.0a0, but we have pyproj with a dependent proj >=8.2.1). 
+  # Generaly, it's not clear if geopandas 0.12.2 has support for python 3.10 on all platfroms,
+  # see https://github.com/geopandas/geopandas/blob/v0.12.2/.github/workflows/tests.yaml
+  skip: True  # [py<38 or s390x or py>=310]
 
 outputs:
   - name: geopandas-base
-    script: build_base.sh
-    build:
-      noarch: python
+    script: build_base.sh   # [unix]
+    script: build_base.bat  # [win]
     requirements:
       host:
-        - python >=3.6
+        - python
         - pip
+        - setuptools
+        - wheel
       run:
-        - python >=3.6
-        - pandas >=0.24
-        - shapely
-        - pyproj >=2.2.0
+        - python
+        - packaging
+        - pandas >=1.0.5
+        - pyproj >=2.6.1.post1
+        - shapely >=1.7
 
     test:
       imports:
         - geopandas
         - geopandas.io
+        - geopandas.tools
+        - geopandas.datasets
       files:
         - test_data
 
   - name: geopandas
     requirements:
       host:
-        - python >=3.6
+        - python
+        - pip
+        - setuptools
+        - wheel
       run:
-        - python >=3.6
+        - python
         - {{ pin_subpackage('geopandas-base', exact=True) }}
-        - fiona
-        - rtree
+        - fiona >=1.8
+        - folium
+        - mapclassify >=2.4.0
         - matplotlib-base
```

```suggestion
        - matplotlib-base >=3.2.0
```
See: https://geopandas.org/en/v0.12.2/getting_started/install.html#dependencies